src/llanalyzer/Defines.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <cstdint>
+
+namespace zeek::llanalyzer {
+
+	using identifier_t = uint32_t;
```

Comment:
Does this need its own header?

Reply:
Not necessarily but what would be good style?

Reply:
If possible include it into a one of the other headers, like the `Analyzer.h`? But not sure if this was introduced because of some cyclic dependency that's makes that tricky.

Reply:
The header was created because of some other types/constants that were in there before. While those were slowly whittled away and moved to other places, this one never got changed. It can definitely move somewhere else. The trick is just finding somewhere good that wouldn't involve over-including a file in places where just the type is needed. 

That said, is there any reason why this type is even needed? Any reason it can't just be uint32_t throughout the code?

Reply:
I think the alias was created when things have been in the flow. If no one expects to encounter use cases that will require other types, we can stick to `uint32_t`. There is a small chance that we might need a more complex type for some use cases. Some protocols define ranges for their identifiers. For example, Ethernet: https://en.wikipedia.org/wiki/Ethernet_frame#Types Right now we return magic values to differentiate, which I don't really like. However, we can also handle these cases by providing a method that allows to call the next packet analyzer from within an analyzer.